autojustice.lua

```diff
@@ -0,0 +1,297 @@
+-- Auto Justice
+--@enable = true
+
+local usage = [[
+Usage
+-----
+enable autojustice
+disable autojustice
+]]
```

Comment:
docs should go in `docs/autojustice.rst`. Copy one of the other files in that directory, like `docs/autofish.rst` to get the structure.